### DISCUSSION TITLE
Temporary mode results in message "you do not have access to this resource", even though chat does work

### DISCUSSION THREAD

**Discussion by xaptronic** [open-webui/open-webui#4916] [Q&A]
When I enable temporary mode from the model drop down and type a message, I get a valid response back - however I also get a message that says "You do not have permission to access this resource. Please contact your administrator for assistance."

![image](https://github.com/user-attachments/assets/f3ba28b1-0031-457a-bc6b-8adffac9d288)

It appears there is an API request being sent to `/api/v1/chats/local` - which invokes code that looks for a chat ID + user combo. However there is no chat ID called `local` paired with any user ID.

Is this a known issue, or is it possibly somehow to do with how I have things setup?

**Comment by silentoplayz** [comment]
I can confirm that this is happening on the latest dev branch and is easily reproducible by toggling on temporary chat from within the model dropdown selector and sending a query.

**Reply by lehhair** [reply]
I've encountered this as well, in addition I've found that if the administrator doesn't whitelist the models used for the panels for the user, it's impossible to generate a title for the user because the user doesn't have that permission, but that's probably correct

**Comment by AndyX-Net** [comment]
same issue, docker environment, v0.3.16 - 2024-08-27

**Comment by tjbck** [comment]
Should be fixed on dev!

**Reply by silentoplayz** [reply]
Confirming that it is indeed fixed on the dev branch!

**Comment by TheTechromancer** [comment]
Running into this when using Microsoft OAUTH. Everything works fine until after signing in, then on the final redirect to `/oauth/microsoft/callback`, the server returns a 403:

![image](https://github.com/user-attachments/assets/28bd56b6-16e7-4abe-961e-70421d779b08)

Server log:
![image](https://github.com/user-attachments/assets/c52ebe07-1d66-4f16-af32-37e862773924)

**Reply by TheTechromancer** [reply]
Forgot to set `ENABLE_OAUTH_SIGNUP=true`

**Reply by arunbugkiller** [reply]
@TheTechromancer I have set ENABLE_OAUTH_SIGNUP=TRUE, still getting the same error

**Reply by ChandanTeekinavar** [reply]
> Forgot to set `ENABLE_OAUTH_SIGNUP=true`

I was also having same problem with OIDC, It worked me by enabling ENABLE_OAUTH_SIGNUP

**Reply by mzyil** [reply]
@arunbugkiller  I had set it as `ENABLE_OAUTH_SIGNUP=1`, changing it to `ENABLE_OAUTH_SIGNUP=true` worked
